examples/meta/src/statistical_testing/quadratic_time_maximum_mean_discrepancy.sg

```diff
@@ -56,7 +56,7 @@ int num_runs = 1
 int num_folds = 3
 mmd.set_kernel_selection_strategy(enum EKernelSelectionMethod.KSM_CROSS_VALIDATION, num_runs, num_folds, alpha)
 mmd.select_kernel()
-Kernel learnt_kernel_single = mmd.get_kernel()
```

Comment:
@lambday here is an example of why the testing framework needs refactoring. I keep on having to do such hacks